devel/libecl/src/smspec_node.c

```diff
@@ -736,6 +736,38 @@ smspec_node_type * smspec_node_alloc_lgr( ecl_smspec_var_type var_type ,
   }
 }
 
+smspec_node_type* smspec_node_copy( const smspec_node_type* node ) {
```

Comment:
I'll change the name, that's perfectly fine. I apologise for my lack of familiarity with ert here, obviously I should've studied it better.